content/en/error_tracking/guides/enable_apm.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Enable APM
+further_reading:
+- link: "/tracing"
+  tag: "Documentation"
+  text: "Learn about APM"
+- link: "/error_tracking/guides/enable_infra"
+  tag: "Guide"
+  text: "Enable Infrastructure Monitoring"
+---
+
+This guide explains how to update the Datadog Agent configuration to enable [APM][1].
```

Comment:
Can you add 1-2 sentences about the benefits of enabling APM with backend error tracking?

Reply:
Would that answer your request?

```suggestion
[Datadog Application Performance Monitoring (APM)][1] provides deep visibility into your applications, enabling you to identify performance bottlenecks, troubleshoot issues, and optimize your services. To leverage these features on top of standalone backend error tracking, update the Datadog Agent configuration to enable APM.
```

Reply:
I updated your suggestion slightly, but yes it's great!